adapter/events.go

```diff
@@ -22,15 +22,26 @@ import (
 
 // Info is used by events to wrap user and provider info.
 type Info struct {
-	User     *UserInfo
 	Provider *ProviderInfo
 }
 
+type EventType string
+
+const (
+	EventChannelMessage      = EventType("channel_message")
```

Comment:
nit: The syntax:
```
EventChannelMessage EventType = "channel_message"
```
seems cleaner. What do you think?

Reply:
Sounds good, done.